UI/data/locale/en-US.ini

```diff
@@ -1049,6 +1049,12 @@ Basic.Settings.Advanced.FormatWarning10BitSdr="Warning: 10-bit formats are more
 Basic.Settings.Advanced.FormatWarning2100="Warning: Rec. 2100 should use a format with more precision."
 Basic.Settings.Advanced.Audio.BufferingTime="Audio Buffering Time"
 Basic.Settings.Advanced.Video.ColorFormat="Color Format"
+Basic.Settings.Advanced.Video.ColorFormat.NV12="NV12 (8 bits, 4:2:0, 2 planes)"
```

Comment:
After some internal discussion, please change "8 bits" and "10 bits" to "8-bit" and "10-bit" respectively.

Reply:
Ok will change in the next update